src/game/Spells/SpellAuras.cpp

```diff
@@ -2062,8 +2072,18 @@ void Aura::HandleAuraDummy(bool apply, bool Real)
             case 23183:                                     // Mark of Frost
             {
                 if (m_removeMode == AURA_REMOVE_BY_DEATH)
-                    target->CastSpell(target, 23182, true, nullptr, this);
-                    return;
+                {
+                    if (Unit* caster = GetCaster())
+                    {
+                        // Azuregos
+                        // Only cast Mark of Frost on targets nearby when engaged
```

Comment:
Is this blizzlike?

Reply:
I also have no poof for this - needs to be confirmed.